### PR TITLE
label-live 3.2.3 (new cask)

### PR DESCRIPTION
**Pull request by dueyfinster**
Added Label LIVE which is a label design
  and printing software for thermal printers.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/l/label-live.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Documentation: https://docs.brew.sh/Cask-Cookbook
+#                https://docs.brew.sh/Adding-Software-to-Homebrew#cask-stanzas
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (bevanjkay) [inline Casks/l/label-live.rb:3]:
```suggestion
```

File: Casks/l/label-live.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+cask "label-live" do
+  version "3.2.3"
+  sha256 "e5cf6eb208c37982afb5452160cd0d5d0380b7da7edf6b8ee7b629709ea91bf1"
+
+  url "https://download.label.live/Label-LIVE-#{version}.dmg"
+  name "Label LIVE"
+  desc "Label Design and Printer Software. Design in minutes. Print in seconds"
```
Comment (bevanjkay) [inline Casks/l/label-live.rb:10]:
```suggestion
  desc "Label design and printer software"

File: Casks/l/label-live.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  homepage "https://label.live/"
+
+  # Documentation: https://docs.brew.sh/Brew-Livecheck
```
Comment (bevanjkay) [inline Casks/l/label-live.rb:13]:
```suggestion
```

File: Casks/l/label-live.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  livecheck do
+    url "https://label.live/thank-you"
+    regex(/"VERSION":"(\d+(?:\.\d+)*)"/i)
+    strategy :page_match
```
Comment (bevanjkay) [inline Casks/l/label-live.rb:17]:
```suggestion
```

The `page_match` strategy can be omitted as it is the default.

File: Casks/l/label-live.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+  url "https://download.label.live/Label-LIVE-#{version}.dmg"
+  name "Label LIVE"
+  desc "Label Design and Printer Software. Design in minutes. Print in seconds"
+  homepage "https://label.live/"
+
+  # Documentation: https://docs.brew.sh/Brew-Livecheck
+  livecheck do
+    url "https://label.live/thank-you"
+    regex(/"VERSION":"(\d+(?:\.\d+)*)"/i)
```
Comment (bevanjkay) [inline Casks/l/label-live.rb:16]:
```suggestion
    regex(/"VERSION"\s*:\s*"(\d+(:?\.\d+)+)"/i)
```

A minor revision to the regex to align with our standard in this scenario.

File: Casks/l/label-live.rb [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    strategy :page_match
+  end
+
+  # Based on MAS listing: https://apps.apple.com/ie/app/label-live/id1438785281?mt=12
```
Comment (bevanjkay) [inline Casks/l/label-live.rb:20]:
```suggestion
```

This comment can be omitted, as `brew audit` will cross-reference the `depends_on` value with what is found in the app's `info.plist` file.

File: Casks/l/label-live.rb [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  depends_on macos: ">= :high_sierra"
+
+  app "Label LIVE.app"
+
+  # Documentation: https://docs.brew.sh/Cask-Cookbook#stanza-zap
```
Comment (bevanjkay) [inline Casks/l/label-live.rb:25]:
```suggestion
```

**Review by bevanjkay — COMMENTED** [review]
@dueyfinster Thank you for your contribution to `homebrew-cask` - I have left some comments in my review.

File: Casks/l/label-live.rb (hunk 7 above)
Comment (dueyfinster) [reply Casks/l/label-live.rb]: removed thanks

File: Casks/l/label-live.rb (hunk 6 above)
Comment (dueyfinster) [reply Casks/l/label-live.rb]: Great to know - thank you - have removed

File: Casks/l/label-live.rb (hunk 5 above)
Comment (dueyfinster) [reply Casks/l/label-live.rb]: Thank you - updated now

File: Casks/l/label-live.rb (hunk 4 above)
Comment (dueyfinster) [reply Casks/l/label-live.rb]: Thanks - have remove

File: Casks/l/label-live.rb (hunk 2 above)
Comment (dueyfinster) [reply Casks/l/label-live.rb]: updated thanks

File: Casks/l/label-live.rb (hunk 1 above)
Comment (dueyfinster) [reply Casks/l/label-live.rb]: wow I should have caught this - thanks !

File: Casks/l/label-live.rb [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  sha256 "e5cf6eb208c37982afb5452160cd0d5d0380b7da7edf6b8ee7b629709ea91bf1"
+
+  url "https://download.label.live/Label-LIVE-#{version}.dmg"
+  name "Label LIVE"
+  desc "Label design and printer software"
+  homepage "https://label.live/"
+
+  livecheck do
+    url "https://label.live/thank-you"
+    regex(/"VERSION"\s*:\s*"(\d+(:?\.\d+)+)"/i)
```
Comment (khipp) [inline Casks/l/label-live.rb:12]:
```suggestion
    url "https://s3.amazonaws.com/labellive.semireg.com/latest-mac.yml"
    strategy :electron_builder
```

File: Casks/l/label-live.rb [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  depends_on macos: ">= :high_sierra"
+
+  app "Label LIVE.app"
+
+  zap trash: [
+    "~/Library/Application Support/Label LIVE",
+    "~/Library/Logs/Label LIVE",
+    "~/Library/Saved Application State/com.semireg.LabelLIVE.savedState",
```
Comment (khipp) [inline Casks/l/label-live.rb:22]:
```suggestion
    "~/Library/Application Support/Label LIVE",
    "~/Library/Logs/Label LIVE",
    "~/Library/Preferences/com.semireg.LabelLIVE.plist",
    "~/Library/Saved Application State/com.semireg.LabelLIVE.savedState",
```

**Review by khipp — APPROVED** [review]
Thank you!

**Comment by dueyfinster** [comment]
Thanks for changes and merging @khipp ! 👍

**Pull request merged**